frontend/src/App.svelte

```diff
@@ -8,11 +8,11 @@
 </script>
 
 <Router>
-	<Route path="/">
-		<h1>Welcome to the Open Source Raid Guild</h1>
-	</Route>
-	<Route path="quests" component={QuestBoard} />
-	<Route path="raids" component={Raids} />
-	<Route path="raids/:id" component={ViewRaid} />
-	<Route path="*" component={NotFound} />
+  <Route path="/">
+    <h1>Welcome to the Open Source Raid Guild</h1>
+  </Route>
+  <Route path="quests" component={QuestBoard} />
+  <Route path="raids" component={Raids} />
+  <Route path="raids/:id" component={ViewRaid} />
+  <Route path="*" component={NotFound} />
```

Comment:
Ah. Tab spacing changed or something. VSCode didn't show any diffs :sweat_smile:

Reply:
Yeah, I figured more specific Prettier rules might be necessary 😂

Reply:
Oh remove the ESLint rule for tabs and just let Prettier handle it